hbase-server/src/test/java/org/apache/hadoop/hbase/replication/TestSyncReplicationStandbyKillRS.java

```diff
@@ -72,16 +72,26 @@ public void testStandbyKillRegionServer() throws Exception {
     Thread t = new Thread(() -> {
       try {
         List<JVMClusterUtil.RegionServerThread> regionServers =
-            UTIL2.getMiniHBaseCluster().getLiveRegionServerThreads();
+          UTIL2.getMiniHBaseCluster().getLiveRegionServerThreads();
+        LOG.debug("Going to stop {} RSes: [{}]",
+          regionServers.stream().map(rst -> rst.getRegionServer().getServerName().getServerName())
+            .collect(Collectors.joining(", ")));
```

Comment:
No of placeholders (2) are not matching no of actual arguments (1)

Reply:
Thanks for pointing this out.

Fixed.